docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -156,6 +156,10 @@ Users can install and run multiple versions of the .NET Framework on their compu
 
 - The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed (returning `True` if it is and `False` otherwise). You can substitute another value listed in the table to check for a different minimum-required .NET Framework version.
 
+```PowerShell
+Get-ChildItem "hklm:SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\" | Get-ItemPropertyValue -Name Release | % { $_ -ge 394802 } 
+```
+
```

Comment:
Addressed in commit https://github.com/dotnet/docs/commit/7394864f310d22ca00e75053bbe8eae37b901f65. The fix was to indent the PowerShell snippet.